org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/CachingCompiler.scala

```diff
@@ -0,0 +1,75 @@
+import java.io.File
+
+import org.scalaide.core.internal.builder.zinc.SbtInputs
+import org.scalaide.util.internal.SbtUtils
+
+import sbt.CompileSetup
+import sbt.EmptyAnalysis
+import sbt.compiler.AnalyzingCompiler
+import sbt.compiler.IC
+import sbt.compiler.MixedAnalyzingCompiler
+import sbt.inc.Analysis
+import sbt.inc.AnalysisStore
+import xsbti.Logger
+import xsbti.Reporter
+import xsbti.compile.Compilers
+
+package org.scalaide.core.internal.builder.zinc {
```

Comment:
Please add the package declaration at the top and move the EmptyAnalysis file to its own file. It is annoying if one needs to find classes, which can't easily be found through the file system.